lib/token/symmetric.go

```diff
@@ -0,0 +1,194 @@
+package token
+
```

Comment:
Uhm, can we just use the crypto blowfish library here instead of symmetric encryption? seems like it's more standard